Microsoft.Toolkit/Extensions/ArrayExtensions.cs

```diff
@@ -13,71 +14,6 @@ namespace Microsoft.Toolkit.Extensions
     /// </summary>
     public static class ArrayExtensions
     {
-        /// <summary>
-        /// Fills elements of a rectangular array at the given position and size to a specific value.
-        /// Ranges given will fill in as many elements as possible, ignoring positions outside the bounds of the array.
-        /// </summary>
-        /// <typeparam name="T">The element type of the array.</typeparam>
-        /// <param name="array">The source array.</param>
-        /// <param name="value">Value to fill with.</param>
-        /// <param name="row">Row to start on (inclusive, zero-index).</param>
-        /// <param name="col">Column to start on (inclusive, zero-index).</param>
-        /// <param name="width">Positive width of area to fill.</param>
-        /// <param name="height">Positive height of area to fill.</param>
-        public static void Fill<T>(this T[,] array, T value, int row, int col, int width, int height)
```

Comment:
@Sergio0694 thinking same story here as for the HP package, maybe we want to obsolete it to point to the HP package first? I think the main usage is in UniformGrid, but just want to be careful here too?

Reply:
I had done that originally but then switched to just removing them entirely following your comment here: https://github.com/windows-toolkit/WindowsCommunityToolkit/issues/3062#issuecomment-678397811. Let me know how you'd like me to proceed, I can revert these and mark them as obsolete if you want. I'm just curious, any particular reason why you changed your mind on this? Just to make the transition easier for devs? 🙂

Reply:
Yeah, this is ok then, we just need to call it out in the release notes. We should probably open up a doc PR that shows folks how to migrate their code from these to the HP ones? I wonder if we want a general 7.0 migration guide doc vs. individual articles for each thing? Thoughts?

To clarify the HP equivalents are already supported, eh? The #3353 just expands on them, correct?

Reply:
> We should probably open up a doc PR that shows folks how to migrate their code from these to the HP ones? I wonder if we want a general 7.0 migration guide doc vs. individual articles for each thing?

Mmh I'm thinking maybe a single migration doc with various sections grouping changes in each package would be enough? Not sure we'd have enough breaking changes in each separate package to fill an entire separate docs page, and maybe having a single one would make it easier for devs to glance over all the various changes for the whole release? 🤔

> To clarify the HP equivalents are already supported, eh? The #3353 just expands on them, correct?

Yes, there are a bunch of unrelated new APIs, but all these have a direct replacement in the HP package 😊

Reply:
Cool, yeah. We should probably just have a `7.0 migration guide` doc. Now that we have the wiki being setup, it is something we could place there instead of on our main docs page. For now let's move ahead.